doc/extensions/authoring.md

```diff
@@ -112,8 +112,7 @@ See [Extension Metadata](metadata.md) for more information.
 ### Limit dependencies in setup.py
 
 - Before adding a dependency to your setup.py, check that it's not already available in [azure-cli-core setup.py](https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/setup.py).
-- For Azure SDKs, use autorest to generate your SDK into a package that isn't under the `azure` directory.
-- You can verify that your extension doesn't use the `azure` directory by opening your `.whl` and opening the `top_level.txt` file in the `*.dist-info` directory. It should not contain `azure`.
+- Azure SDK or Azure Management SDK dependencies may be overridden by the versions installed as requirements of azure-cli-core. If you use any, test carefully, gracefully handle API changes, and be prepared to release updates.
```

Comment:
```suggestion
- For Azure SDKs, suggest to use autorest to generate your SDK into a pacakage that isn't under the `azure`  directory. Azure SDK or Azure Management SDK dependencies may be overridden by the versions installed as requirements of azure-cli-core. If you use any, test carefully, gracefully handle API changes, and be prepared to release updates.
```

Reply:
Added a slightly different comment with the same intend.